twoliter/embedded/build.Dockerfile

```
@@ -108,6 +108,10 @@ RUN --mount=target=/host \
       -exec cp {} ./rpmbuild/SOURCES/ \; && \
     rm /bypass
 
+RUN --mount=source=advisories,target=/home/builder/advisories \
+    --mount=target=/host \
+    /host/build/tools/advisory-checker --spec-file rpmbuild/SPECS/${PACKAGE}.spec --advisories-dir /home/builder/advisories/
+
```

I don't think this will this work for custom kits that may not include an `advisories` directory. Probably safer to mark as not required in the mount args and check for existence of the advisories dir:

```suggestion
RUN --mount=source=advisories,target=/home/builder/advisories,required=false \
    --mount=target=/host \
    if [ -d /home/builder/advisories ]; then \
        /host/build/tools/advisory-checker --spec-file rpmbuild/SPECS/${PACKAGE}.spec --advisories-dir /home/builder/advisories/ \
    fi
```

Alternatively, could just set `required=false` in the mount args and in your advisory-checker binary, throw a warning if the dir doesn't exist and move on.

I'd also advocate for unit/integration testing for this case.

---

The docker version we have doesn't support `required`. I handled it by mounting the host (the entire kit) directory instead of just advisories and checking the presence of advisories directory in the binary itself.